packages/babel/src/transformation/file/index.js

```diff
@@ -106,7 +107,9 @@ export default class File {
 
 
   /**
-   * [Please add a description.]
+   * Initialize this.opts (construct, normalize, set defaults).
+   * @param object opts Plain options hash passed in.
+   * @return OptionManager
```

Comment:
> `initOptions` returns `Object` not `OptionManager`.

Sorry about that. I tried to rush through this too much and obviously was too sloppy. I'll fix that along with other problems